src/app/Scenes/Home/Components/HeroUnitsRail.tsx

```diff
@@ -0,0 +1,121 @@
+import { Box, Flex, Spacer, Text, Button, Touchable } from "@artsy/palette-mobile"
+import {
+  HeroUnitsRail_heroUnitsConnection$data,
+  HeroUnitsRail_heroUnitsConnection$key,
+} from "__generated__/HeroUnitsRail_heroUnitsConnection.graphql"
+import OpaqueImageView from "app/Components/OpaqueImageView/OpaqueImageView"
```

Comment:
One suggestion here, lets use the OpaqueImageView2 here since its the new image component that we use on eigen (on vscode this might be more easily detectable since the other image component is marked as deprecated)

```suggestion
import { OpaqueImageView } from "app/Components/OpaqueImageView2"
```

Reply:
Done! 🙌

Reply:
Use the `Image` component from Palette!

Reply:
Oh good point! Is it ready to be used everywhere @damassi ?

Reply:
Yep, seems good to go! If there's something that's missing we should add it over there to be used here, but seems fine as is right now.

Reply:
Amazing!

Reply:
https://github.com/artsy/eigen/pull/8946 opened this to mark it as deprecated and point people to use the new one 🌟

Reply:
Ok got it - moved to using palette mobile's Image component 👍